src/packages/core/src/options/server-options.ts

```diff
@@ -1,5 +1,6 @@
 import { DefaultFlavor, FilecoinFlavorName } from "@ganache/flavors";
 import { Definitions } from "@ganache/options";
+import { kMaxLength } from "buffer";
```

Comment:
A Buffer's maximum length allowed by node may differ between versions, architectures, and OSes, so we can use this value to get the maximum length allowed for single buffer instance.